lib/_stream_writable.js

```diff
@@ -307,7 +303,12 @@ function writeOrBuffer(stream, state, chunk, encoding, cb) {
 
   if (state.writing || state.corked) {
     var last = state.lastBufferedRequest;
-    state.lastBufferedRequest = new WriteReq(chunk, encoding, cb);
+    state.lastBufferedRequest = {
+      chunk: chunk,
+      encoding: encoding,
+      callback: cb,
+      next: null
+    };
```

Comment:
we can probably reduce a bit the code footprint here by using:

```js
{  chunk, encoding, callback: cb, next: null }
```

Not sure if this functions was inlineable before and/or this change remove that.